pkg/asset/manifests/template.go

```diff
@@ -57,6 +57,7 @@ type cloudCredsSecretData struct {
 }
 
 type bootkubeTemplateData struct {
+	CloudCredentialsMode       string
```

Comment:
this looks like left over from move from template to install config on bootstrap-host...

Reply:
Yes. The changes to this file have been removed now.